samples/mentions.html

```diff
@@ -0,0 +1,273 @@
+<!DOCTYPE html>
+<!--
+	Copyright (c) 2003-2018, CKSource - Frederico Knabben. All rights reserved.
+	For licensing, see license.html or https://sdk.ckeditor.com/license.html.
+-->
+<html>
+	<head>
+		<meta charset="utf-8">
+		<meta name="description" content="Mentions plugin">
+		<meta name="keywords" content="ckeditor, editor, wysiwyg, mentions, complete, autocomplete, default, configure, configuration, setup, settings, options, customization, customize, customise, customisation, config, modification, modify, change">
+		<meta name="sdk-samples" content="Mentions plugin">
+		<title>Mentions plugin</title>
+		<link href="https://fonts.googleapis.com/css?family=Maven+Pro:700,500,400" rel="stylesheet">
+		<link href="../template/theme/css/sdk.css" rel="stylesheet">
+		<script src="../../ckeditor-dev/ckeditor.js"></script>
+		<script src="assets/picoModal-2.0.1.min.js"></script>
+		<script src="assets/contentloaded.js"></script>
+		<script src="assets/simplesample.js"></script>
+		<script src="assets/beautify-html.js"></script>
+		<script src="assets/sample.js"></script>
+		<!--[if lt IE 9]>
+			<script src="assets/html5shiv.min.js"></script>
+		<![endif]-->
+		<link rel="icon" href="../template/theme/img/favicon.ico">
+	</head>
+	<body>
+		<style data-sample="1">
+		.columns {
+			background: #fff;
+			padding: 20px;
+			border: 1px solid #E7E7E7;
+		}
+		.columns:after {
+			content: "";
+			clear: both;
+			display: block;
+		}
+		.editor {
+			float: left;
+			width: 55%;
+			position: relative;
+			z-index: 1;
+		}
+		.chat {
+			float: right;
+			width: 40%;
+			box-sizing: border-box;
+			border: 1px solid #E7E7E7;
+		}
+		.chat-input {
+			border: 1px solid #E7E7E7;
+		}
+		.chat-send {
+			cursor: pointer;
+			text-align: center;
+			padding-bottom: 5px;
+		}
+		.comments {
+			height: 550px;
+			list-style-type: none;
+			margin: 0 !important;
+			padding: 0;
+			overflow-y: auto;
+		}
+		.comments li {
+			background: #FAFAFA;
+			margin-bottom: 1px;
+			padding: 5px;
+		}
+		.comments li:nth-child(2n) {
+			background: #F3F3F3;
+		}
+		.comments li:hover {
+			background: #FFFDE3;
+		}
+		.message {
+			padding: 0 10px;
+		}
+		.contact .u-photo {
+			display: inline-block;
+			vertical-align: middle;
+			margin-right: 10px;
+		}
+		.sdk-contents .cke {
+			margin: 0;
+		}
+		</style>
+
+		<header class="sdk-header">
+		</header>
+		<section class="sdk-container">
+			<nav class="sdk-sidebar">
+			</nav>
+			<section class="sdk-contents">
+				<h1>Mentions plugin</h1>
+
+				<p>
+				The mentions plugin has been introduced in CKEditor 4.10 and it provides smart autocompletion feature for custom text matches based on user input. Every time when user type selected marker he will get information about available, existing options.
+				</p>
+
+				<p>Mentions plugin allows you to use different data sources:</p>
```

Comment:
This section, together with the following list is not needed here - that's a bit too technical. This kind of info should be available in ckeditor-docs.